include/cuda/std/barrier

```diff
@@ -671,6 +676,29 @@ struct __memcpy_async_impl<16, false> {
 template<std::size_t _Alignment>
 struct __memcpy_async_impl<_Alignment, true> : public __memcpy_async_impl<16, false> { };
 
+struct __memcpy_async_ublk_impl {
+    template<typename _Sync>
+    __device__ static inline async_contract_fulfillment __copy(_Sync & __sync, char * __destination, char const * __source, std::size_t __size) {
```

Comment:
This is currently only used in `__memcpy_async_sm_dispatch` with the following signature:
```cpp
__memcpy_async_sm_dispatch(_Group const & __group, 
                           char * __destination, char const * __source,
                           std::size_t __size, 
                           _Sync & __sync, 
                           async_contract_fulfillment & __is_async)
```

However, here the function signature is
```cpp
__copy(_Sync & __sync, 
       char * __destination, 
       char const * __source, 
       std::size_t __size)
```

I would prefer if we move `_Sync & __sync` to the end of the argument list to match the other

Reply:
All good suggestions, will add them